nixos/modules/services/network-filesystems/kubo.nix

```
@@ -211,6 +232,28 @@ in
   ###### implementation
 
   config = mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = !builtins.hasAttr "Identity" cfg.settings;
+        message = ''
+          You can't set services.kubo.settings.Identity.
+          Setting services.kubo.settings.Identity.PeerID would
+          break the Kubo configuration and setting
+          services.kubo.settings.Identity.PrivKey would prevent
+          the daemon from starting with
+          "Error: setting private key with API is not supported".
```

```suggestion
          You can't set services.kubo.settings.Identity. Setting services.kubo.settings.Identity.PeerID would break the Kubo configuration
          and setting services.kubo.settings.Identity.PrivKey would prevent the daemon from starting with "Error: setting private key with API is not supported".
```

More important than my formatting change is why? Thats the most interesting part for people and makes them better understand this.